src/App.tsx

```diff
@@ -1,28 +1,78 @@
 import React from 'react';
 
-const App = () => {
+const App:React.FC = () => {
 
-  var tasks : string[] = ['Task 1', 'Task 2'];
+  var tasks: string[] = ['Task 1', 'Task 2'];
 
-  const [task , setTask] = React.useState<Array<string>>();
+ // type AddedTask = string;
 
- 
+  const [allTask , setAllTask] = React.useState<string[]>(tasks);
 
+  const [inputValue, setInputValue] = React.useState<string>('');
+
+  
+  const handleTask = () => {
+    setAllTask([...allTask, inputValue]);
+  }
+
+  const handleInput = event => {
+    setInputValue(event.target.value);
+  }
 
   return (
-    <div>
-       <h1>To Do List </h1><br/>
-       <h2>Tasks</h2>
-       <AllTask task={tasks} />
+    <div className="App">
+      <h1>To Do App </h1><hr />
+      <h2>Add Task</h2>
+      <input type="text" value={inputValue} onChange = {handleInput} /><br/>  
+          <button type="button" onClick={handleTask}>Add Task</button>
+      <hr />
+      <h2>All Tasks</h2>
+      <AllTask task={allTask} />
     </div>
   )
 }
 
+type TaskProp = {
+  task: string[];
```

Comment:
rename this variable .
```suggestion
  allTask: string[];
```